src/fs/syslib/kfsop.c

```diff
@@ -347,8 +347,29 @@ int kformat(const char *pathname, const char *fs_type) {
 	return 0;
 }
 
-int kmount(const char *dev, const char *dir, const char *fs_type) {
-	struct path dev_node, dir_node, root_path;
+struct block_dev *bdev_by_path(const char *source) {
+	struct path dev_node;
+	const char *lastpath;
+	int res;
+
+	if (source == NULL) {
+		return NULL;
+	}
+
+	if (ENOERR != (res = fs_perm_lookup(source, &lastpath, &dev_node))) {
+		return NULL;
+	}
+
+	if (ENOERR != (res = fs_perm_check(dev_node.node, S_IROTH | S_IXOTH))) {
+		return NULL;
+	}
```

Comment:
`res` is not used so we can remove this variable.

Reply:
Done